### ISSUE TITLE
:caption: flag in graphviz directives causes the HTML writer to crash

### ISSUE THREAD

**Issue by pchampin** [sphinx-doc/sphinx#1917]
Steps to reproduce the bug:
1. create a fresh Sphinx project
2. include sphinx.ext.graphviz in the extensions
3. add the text below in `index.rst`:
   
   ```
   .. digraph:: foo
       :caption: the caption of my graph
   
       a -> b
   ```
4. run `make html`

With Sphinx 1.3.1, I get the following exception:

```
Exception occurred:
  File "/home/pa/.local/lib/python2.7/site-packages/sphinx/writers/html.py", line 353, in depart_caption
      node.parent.traverse(nodes.image)[0], _('Permalink to this image'))
IndexError: list index out of range
```

**Comment by andreacassioli** [comment]
I confirm the issue on Sphinx 1.3.1

**Comment by tk0miya** [comment]
Sorry for inconvinience.
but it is already fixed (at #1788). please wait for next release.

thanks.
